### PR TITLE
Alert V3 backround and boder color fix

### PR DESCRIPTION
**Pull request by harshil1793**
## Chromatic

https://2339-alert-info-color-fix--60f9b557105290003b387cd5.chromatic.com

---

## Description
This PR fixes the colors for Alert component for both mobile and desktop views.
Closes https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2339

## QA Checklist
- [x] Component maintains 1:1 parity with design mocks
- [x] Text is consistent with what's been provided in the mocks
- [x] Component behaves as expected across breakpoints
- [x] Accessibility expert has signed off on code changes (if applicable. If not applicable provide reason why)
N/A
- [x] Designer has signed off on changes (if applicable. If not applicable provide reason why)
- [x] Tab order and focus state work as expected
N/A
- [x] Changes have been tested against screen readers (if applicable. If not applicable provide reason why)
N/A
- [x] New components are covered by e2e tests; updates to existing components are covered by existing test suite
N/A
- [x] Changes have been tested in vets-website using Verdaccio (if applicable. If not applicable provide reason why)
N/A

## Screenshots
Screenshot 2023-12-21 at 10 30 33 AM
Screenshot 2023-12-21 at 10 30 48 AM
Screenshot 2023-12-21 at 10 31 06 AM
Screenshot 2023-12-21 at 10 32 28 AM

## Acceptance criteria
- [x] QA checklist has been completed
- [x] Screenshots have been attached that cover desktop and mobile screens

## Definition of done
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

**Review by powellkerry — APPROVED** [review]

File: packages/web-components/src/components/va-alert/va-alert.scss [hunk 1]
```diff
@@ -109,29 +109,29 @@ i.fa-times-circle::before {
 
 // Override USWDS background and border colors for VA styles.
```
Comment (jamigibbs) [inline packages/web-components/src/components/va-alert/va-alert.scss:110]: Can you update this comment or remove it? It doesn't make sense anymore if we're just using USWDS now.

File: packages/web-components/src/components/va-alert/va-alert.scss [hunk 2]
```diff
@@ -109,29 +109,29 @@ i.fa-times-circle::before {
...
 .usa-alert--warning {
-  border-color: var(--color-gold);
+  border-color: var(--uswds-system-color-gold-vivid-20);
```
Comment (jamigibbs) [inline packages/web-components/src/components/va-alert/va-alert.scss:112]: If you remove these USWDS CSS overrides completely from the stylesheet, does the USWDS system just naturally display the correct color?

Comment (jamigibbs) [reply packages/web-components/src/components/va-alert/va-alert.scss:112]: For example, the override here seems like it's just applying the same color from default styles of `.usa-alert--error`:

Screenshot 2023-12-21 at 9 52 01 AM

Comment (harshil1793) [reply packages/web-components/src/components/va-alert/va-alert.scss:111]: Yes, it naturally displayed correct colors.

Comment (harshil1793) [reply packages/web-components/src/components/va-alert/va-alert.scss:111]: Those are coming from css-library, that is how it should be correct?

Comment (jamigibbs) [reply packages/web-components/src/components/va-alert/va-alert.scss:111]: We are importing this module below from the USWDS system and that's applying USWDS styles to `.usa-alert--error` `.usa-alert` etc.

https://github.com/department-of-veterans-affairs/component-library/blob/840b4073909428ab66d4394f1d0afc0ce8162295/packages/web-components/src/components/va-alert/va-alert.scss#L3

It applies the USWDS colors by default. So I'm wondering if we can just delete our VA override CSS completely and let the module do its thing now? I'm not sure that we need to involve the CSS Library tokens at all. 🤷🏻‍♀️

Comment (harshil1793) [reply packages/web-components/src/components/va-alert/va-alert.scss:111]: Absolutely, I agree with your assessment. I've just tried using the module from the USWDS system as you mentioned, and it works perfectly. This confirms that we can indeed remove our VA override CSS and allow the module to apply the USWDS styles and colors by default. It seems that involving the CSS Library tokens might be unnecessary in this context. I'll go ahead and make that change now.

Thanks for pointing this out! 🙌🏼

**Review by jamigibbs — APPROVED** [review]

**Pull request merged**